src/CMakeLists.txt

```diff
@@ -17,30 +17,71 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.4)
```

Comment:
Does it make sense to have a cmake version requirement besides the one in the root CMakeLists.txt?

Reply:
?

Reply:
Dunno, was there before. Any preference?

Reply:
Agreed. We haven't supported 2.8 even in the src directory for quite a long time.